src/csvcubed/writers/skoscodelistwriter.py

```diff
@@ -38,6 +39,15 @@ class SkosCodeListWriter(WriterBase):
     def csv_metadata_file_name(self) -> str:
         return f"{self.csv_file_name}-metadata.json"
 
+    @staticmethod
+    def has_duplicated_qb_concepts(code_list: NewQbCodeList) -> bool:
+        duplicated_qb_concepts: list[DuplicatedQbConcept] = [
```

Comment:
I'd replace this with `return any(True for concept in code_list.concepts if isinstance(concept, DuplicatedQbConcept)` This should hopefully avoid actually creating the list.